### PR TITLE
Feat admin login system

### PR DESCRIPTION
**Pull request by joeybisonai**
# Description
스크린샷 2023-07-04 오후 4 04 36

I created a login system so that user can't access the data if user didn't receive the token.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

- [x] I have performed a self-review of my code.

### CODE REVIEW

File: admin/src/lib/cookies.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// @ts-ignore
```
Comment (martinkersner) [inline admin/src/lib/cookies.ts:1]: Why do we use `@ts-ignore` here?

**Review by martinkersner — APPROVED** [review]
LGTM! I left a couple of comments, but in general looks good to me!

File: admin/src/lib/cookies.ts (hunk 1 above)
Comment (joeybisonai) [reply admin/src/lib/cookies.ts:1]: If @ts-ignore is not here, an error occurs and no import is imported.

File: admin/src/components/Template/Home/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  onLogin: () => void;
+}
+
+export const LoginPage: React.FC<LoginPageProps> = ({ onLogin }) => {
+  const [password, setPassword] = useState("");
+  const { addToast } = useToastContext();
+  const handleLogin = async () => {
+    try {
+      const response = await authenticatedAxios.post(
+        "http://localhost:8888/auth/login",
```
[the remark this answers is not in the document]
Comment (joeybisonai) [reply admin/src/components/Template/Home/Login/index.tsx:30]: Yes, I'll fix it

**Pull request merged**